packages/js/src/classic-editor/components/metabox/index.js

```diff
@@ -0,0 +1,140 @@
+import SeoAnalysis from "../seo-analysis";
+import ReadabilityAnalysis from "../readability-analysis";
+import MetaboxCollapsible from "../../../components/MetaboxCollapsible";
+import SocialMetadataPortal from "../../../components/portals/SocialMetadataPortal";
+import SidebarItem from "../../../components/SidebarItem";
+import AdvancedSettings from "../../../containers/AdvancedSettings";
+import CollapsibleCornerstone from "../../../containers/CollapsibleCornerstone";
+import SchemaTabContainer from "../../../containers/SchemaTab";
+import SEMrushRelatedKeyphrases from "../../../containers/SEMrushRelatedKeyphrases";
+import Warning from "../../../containers/Warning";
+import { EDITOR_STORE_NAME } from "../../editor-store";
+
+import { SeoResultsContainer } from "@yoast/seo-integration";
+import { ContentAnalysis } from "@yoast/analysis-report";
+
+/**
+ * Creates the focus keyphrase input component.
+ *
+ * @param {string} focusKeyphraseInfoLink The URL for the help link.
+ *
+ * @returns {JSX.Element} The focus keyphrase input.
+ */
+const FocusKeyphraseInput = ( { focusKeyphraseInfoLink } ) => {
+	const focusKeyphrase = useSelect( select => select( SEO_STORE_NAME ).selectKeyphrase() );
+	const displayNoKeyphraseMessage = useSelect( select => select( EDITOR_STORE_NAME ).getSEMrushNoKeyphraseMessage() );
+	const displayNoKeyphraseForTrackingMessage = useSelect( select => select( EDITOR_STORE_NAME ).hasWincherNoKeyphrase() );
+	const isSEMrushIntegrationActive = useSelect( select => select( EDITOR_STORE_NAME ).getIsSEMrushIntegrationActive() );
+	const { updateKeyphrase } = useDispatch( SEO_STORE_NAME );
+	const { setMarkerPauseStatus } = useDispatch( EDITOR_STORE_NAME );
+
+	const handleFocusKeyphraseChange = useCallback( keyphrase => updateKeyphrase( { keyphrase } ), [ updateKeyphrase ] );
+
+	const pauseMarker = useCallback( () => setMarkerPauseStatus( true ), [ setMarkerPauseStatus ] );
+	const startMarker = useCallback( () => setMarkerPauseStatus( false ), [ setMarkerPauseStatus ] );
+
+	return (
+		<KeywordInput
+			keyword={ focusKeyphrase }
+			displayNoKeyphraseMessage={ displayNoKeyphraseMessage }
+			displayNoKeyphraseForTrackingMessage={ displayNoKeyphraseForTrackingMessage }
+			isSEMrushIntegrationActive={ isSEMrushIntegrationActive }
+			helpLink={ focusKeyphraseInfoLink }
+			onFocusKeywordChange={ useDebounce( handleFocusKeyphraseChange ) }
+			onFocusKeyword={ pauseMarker }
+			onBlurKeyword={ startMarker }
+		/>
+	);
+};
+
+FocusKeyphraseInput.propTypes = {
+	focusKeyphraseInfoLink: PropTypes.string.isRequired,
+};
+
+/**
+ * Creates the Metabox component.
+ *
+ * @returns {JSX.Element} The Metabox.
+ */
+const Metabox = () => {
+	const settings = useSelect( select => select( EDITOR_STORE_NAME ).getPreferences() );
+	const isKeywordAnalysisActive = useSelect( select => select( EDITOR_STORE_NAME ).getIsKeywordAnalysisActive() );
+	const baseUrl = useSelect( select => select( EDITOR_STORE_NAME ).getBaseUrlFromSettings() );
```

Comment:
We have the permalink in our store. I wrote this weird function within the Google preview container to get the base url from that.
That makes it not needed to pass the `baseUrl` here, from the old source.